test/server/options_impl_test.cc

```diff
@@ -271,7 +271,10 @@ TEST_F(OptionsImplTest, OptionsAreInSyncWithProto) {
   // 5. use-fake-symbol-table - short-term override for rollout of real symbol-table implementation.
   // 6. hot restart version - print the hot restart version and exit.
   // 7. log-format-prefix-with-location - short-term override for rollout of dynamic log format.
-  EXPECT_EQ(options->count() - 7, command_line_options->GetDescriptor()->field_count());
+  //
+  // The two deprecated options  "max_stats", "max_obj_name_len" need to be subtracted from
+  // the protobuf field count.
```

Comment:
Per discussion can you just delete and reserve them to fix this?

Reply:
Current proto is:
```
  reserved 12, 20, 21;
  reserved "max_stats", "max_obj_name_len";
```
Based on the v2 stats, those were 20 and 21 so I think I just need to remove the keyword references. I'll try that.

Reply:
I've got test/server:options_impl_test to pass without modifying through a series of unprincipled experiments.